app/helpers/account_helper.rb

```diff
@@ -0,0 +1,5 @@
+module AccountHelper
```

Comment:
🤔 wonder if some kind of shared feature flags should go in the personalisation Gem?